public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -13,12 +13,13 @@ import { PluginDashboards } from '../../PluginDashboards';
 type Props = {
   plugin: CatalogPlugin;
   queryParams: UrlQueryMap;
+  defaultPageId: string;
```

Comment:
**Question:** could this be the type of `PluginTabIds` by any chance?

Reply:
Maybe I'm wrong but I don't think it can be because if a plugin defines [only custom config pages](https://github.com/grafana/grafana/pull/41979/files#diff-956df7b39312fe73ca11c69d7cf84d357de3cd8d0dbbd1a643558d297f8fc416R60-R62) we don't know the ID. 🤔

Reply:
Aaa, you right bud! 👍